VirtoCommerce.Storefront/Domain/Cart/CartConverter.cs

```diff
@@ -429,13 +429,13 @@ public static ShoppingCart ToShoppingCart(this cartDto.ShoppingCart cartDto, Cur
                 Customer = user
             };
 
-            if (cartDto.Coupon != null)
+            if (cartDto.Coupons != null)
             {
-                result.Coupon = new Coupon
+                result.Coupons = new List<Coupon>(cartDto.Coupons.Select(coupon => new Coupon
```

Comment:
Can be simplified:
`result.Coupons = cartDto.Coupons.Select(coupon => new Coupon { Code = coupon, AppliedSuccessfully = !string.IsNullOrEmpty(coupon) }).ToList();`

Reply:
Followed the same code pattern as for the other 'cartDto' items below. Should we keep it or refactor the population of other lists too?